[2023-09-18T13:47:24.641+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 6.1-monitoring.task2 scheduled__2023-09-17T00:00:00+00:00 [queued]>
[2023-09-18T13:47:24.651+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 6.1-monitoring.task2 scheduled__2023-09-17T00:00:00+00:00 [queued]>
[2023-09-18T13:47:24.652+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2023-09-18T13:47:24.673+0000] {taskinstance.py:1380} INFO - Executing <Task(BashOperator): task2> on 2023-09-17 00:00:00+00:00
[2023-09-18T13:47:24.680+0000] {standard_task_runner.py:57} INFO - Started process 4489 to run task
[2023-09-18T13:47:24.684+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', '6.1-monitoring', 'task2', 'scheduled__2023-09-17T00:00:00+00:00', '--job-id', '263', '--raw', '--subdir', 'DAGS_FOLDER/6.1-monitoring.py', '--cfg-path', '/tmp/tmp07e5n_7w']
[2023-09-18T13:47:24.686+0000] {standard_task_runner.py:85} INFO - Job 263: Subtask task2
[2023-09-18T13:47:24.747+0000] {task_command.py:415} INFO - Running <TaskInstance: 6.1-monitoring.task2 scheduled__2023-09-17T00:00:00+00:00 [running]> on host e891a57bf0af
[2023-09-18T13:47:24.846+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='6.1-monitoring' AIRFLOW_CTX_TASK_ID='task2' AIRFLOW_CTX_EXECUTION_DATE='2023-09-17T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-09-17T00:00:00+00:00'
[2023-09-18T13:47:24.848+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-09-18T13:47:24.850+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', "sleep 2 && echo 'Task 2"]
[2023-09-18T13:47:24.866+0000] {subprocess.py:86} INFO - Output:
[2023-09-18T13:47:24.868+0000] {subprocess.py:93} INFO - /bin/bash: -c: line 1: unexpected EOF while looking for matching `''
[2023-09-18T13:47:24.870+0000] {subprocess.py:93} INFO - /bin/bash: -c: line 2: syntax error: unexpected end of file
[2023-09-18T13:47:24.871+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2023-09-18T13:47:24.886+0000] {taskinstance.py:1935} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2023-09-18T13:47:24.891+0000] {taskinstance.py:1398} INFO - Marking task as FAILED. dag_id=6.1-monitoring, task_id=task2, execution_date=20230917T000000, start_date=20230918T134724, end_date=20230918T134724
[2023-09-18T13:47:24.908+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 263 for task task2 (Bash command failed. The command returned a non-zero exit code 2.; 4489)
[2023-09-18T13:47:24.938+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-09-18T13:47:24.969+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-09-18T13:53:34.176+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 6.1-monitoring.task2 scheduled__2023-09-17T00:00:00+00:00 [queued]>
[2023-09-18T13:53:34.264+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 6.1-monitoring.task2 scheduled__2023-09-17T00:00:00+00:00 [queued]>
[2023-09-18T13:53:34.265+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2023-09-18T13:53:34.286+0000] {taskinstance.py:1380} INFO - Executing <Task(BashOperator): task2> on 2023-09-17 00:00:00+00:00
[2023-09-18T13:53:34.293+0000] {standard_task_runner.py:57} INFO - Started process 5137 to run task
[2023-09-18T13:53:34.298+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', '6.1-monitoring', 'task2', 'scheduled__2023-09-17T00:00:00+00:00', '--job-id', '420', '--raw', '--subdir', 'DAGS_FOLDER/6.1-monitoring.py', '--cfg-path', '/tmp/tmppkwals3y']
[2023-09-18T13:53:34.300+0000] {standard_task_runner.py:85} INFO - Job 420: Subtask task2
[2023-09-18T13:53:34.365+0000] {task_command.py:415} INFO - Running <TaskInstance: 6.1-monitoring.task2 scheduled__2023-09-17T00:00:00+00:00 [running]> on host e891a57bf0af
[2023-09-18T13:53:34.484+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='6.1-monitoring' AIRFLOW_CTX_TASK_ID='task2' AIRFLOW_CTX_EXECUTION_DATE='2023-09-17T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-09-17T00:00:00+00:00'
[2023-09-18T13:53:34.486+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-09-18T13:53:34.487+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', "sleep 2 && echo 'Task 2'"]
[2023-09-18T13:53:34.502+0000] {subprocess.py:86} INFO - Output:
[2023-09-18T13:53:36.506+0000] {subprocess.py:93} INFO - Task 2
[2023-09-18T13:53:36.508+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-09-18T13:53:36.561+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=6.1-monitoring, task_id=task2, execution_date=20230917T000000, start_date=20230918T135334, end_date=20230918T135336
[2023-09-18T13:53:36.607+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-18T13:53:36.649+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-09-18T13:55:25.706+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 6.1-monitoring.task2 scheduled__2023-09-17T00:00:00+00:00 [queued]>
[2023-09-18T13:55:25.722+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 6.1-monitoring.task2 scheduled__2023-09-17T00:00:00+00:00 [queued]>
[2023-09-18T13:55:25.723+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2023-09-18T13:55:25.750+0000] {taskinstance.py:1380} INFO - Executing <Task(BashOperator): task2> on 2023-09-17 00:00:00+00:00
[2023-09-18T13:55:25.758+0000] {standard_task_runner.py:57} INFO - Started process 5354 to run task
[2023-09-18T13:55:25.763+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', '6.1-monitoring', 'task2', 'scheduled__2023-09-17T00:00:00+00:00', '--job-id', '473', '--raw', '--subdir', 'DAGS_FOLDER/6.1-monitoring.py', '--cfg-path', '/tmp/tmpywfqx8lm']
[2023-09-18T13:55:25.765+0000] {standard_task_runner.py:85} INFO - Job 473: Subtask task2
[2023-09-18T13:55:25.847+0000] {task_command.py:415} INFO - Running <TaskInstance: 6.1-monitoring.task2 scheduled__2023-09-17T00:00:00+00:00 [running]> on host e891a57bf0af
[2023-09-18T13:55:25.966+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='6.1-monitoring' AIRFLOW_CTX_TASK_ID='task2' AIRFLOW_CTX_EXECUTION_DATE='2023-09-17T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-09-17T00:00:00+00:00'
[2023-09-18T13:55:25.969+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-09-18T13:55:25.970+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', "sleep 2 && echo 'Task 2'"]
[2023-09-18T13:55:25.984+0000] {subprocess.py:86} INFO - Output:
[2023-09-18T13:55:27.988+0000] {subprocess.py:93} INFO - Task 2
[2023-09-18T13:55:27.989+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-09-18T13:55:28.018+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=6.1-monitoring, task_id=task2, execution_date=20230917T000000, start_date=20230918T135525, end_date=20230918T135528
[2023-09-18T13:55:28.068+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-18T13:55:28.102+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
